src/test/java/org/fundacionjala/coding/abel/FizzBuzzTest.java

```diff
@@ -0,0 +1,104 @@
+package org.fundacionjala.coding.abel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class FizzBuzzTest {
+
+
+    private FizzBuzz instance;
+    private List<String> list = new ArrayList<String>();
+    private List<String> list2 = new ArrayList<String>();
+
+    /**
+     *
+     */
+    @Before
+    public void before() {
+        instance = new FizzBuzz();
+        list = instance.fizzbuzz();
+        list2 = instance.fizzBuzz2();
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void test() {
+        final int num = 2;
+        assertEquals("Fizz", list.get(num));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void test2() {
+        final int num2 = 4;
+        assertEquals("Buzz", list.get(num2));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void test3() {
+        final int num3 = 14;
+        assertEquals("FizzBuzz", list.get(num3));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "FizzBuzz" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)